### PR TITLE
Avoids direct dependency on NodeJS "stream" API

### PR DESCRIPTION
**Pull request by Tpt**

### CODE REVIEW

File: lib/SparqlJsonBindingsTransformer.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export class SparqlJsonBindingsTransformer extends Transform {
     this.parser = parser;
   }
 
-  public _transform(chunk: any, encoding: string, callback: TransformCallback): void {
+  public _transform(chunk: any, encoding: string, callback: (error?: Error | null, data?: any) => void): void {
```
Comment (Tpt) [inline lib/SparqlJsonBindingsTransformer.ts:16]: `TransformCallback` is not availlable from `@types/readable-stream`

File: lib/SparqlJsonBindingsTransformer.ts [hunk 2]
```diff
@@ -1,10 +1,12 @@
-import {Transform, TransformCallback} from "stream";
 import {SparqlJsonParser} from "./SparqlJsonParser";
 
+// tslint:disable-next-line:no-var-requires
+const Stream = require("readable-stream");
```
Comment (Tpt) [inline lib/SparqlJsonBindingsTransformer.ts:4]: Not nice but there is no proper typescript definitions for `readable-stream` 4.x (`@type/readable-stream` depends on NodeJS buffer API).

Comment (Tpt) [reply lib/SparqlJsonBindingsTransformer.ts]: Done

**Comment by rubensworks** [comment]
Great, thanks!

Do you already require a new release of this? Or are more changes needed?

**Comment by Tpt** [comment]
I would feel better to add webpack tests before if it's fine for you

**Comment by rubensworks** [comment]
Ok, sounds good!

**Pull request merged**